src/Api/Management/Controller/OrganizationController.php

```diff
@@ -51,6 +51,28 @@ public function __construct(ClientInterface $client, ?EntitySerializerInterface
         parent::__construct($client, $entitySerializer);
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function getEntities(): array
+    {
+        $uri = $this->getBaseEndpointUri();
+        $response = $this->getClient()->get($uri);
+        $responseArray = $this->responseToArray($response);
+        // Ignore entity type key from response, ex.: organizations.
+        $responseArray = reset($responseArray);
+        if (ClientInterface::HYBRID_ENDPOINT == $this->client->getEndpoint()) {
+            $entities = [];
+            foreach ($responseArray as $item) {
+                $entities[$item->organization] = $this->load($item->organization);
+            }
+
+            return $entities;
+        }
+
+        return $this->responseArrayToArrayOfEntities($responseArray);
```

Comment:
Can't we just use the following logic instead?

```
if (ClientInterface::HYBRID_ENDPOINT == $this->client->getEndpoint()) {
  // Do the custom stuff and return
}

return parent::getEntities(); // parent is the inherited method from the trait.

```

You may need to alias the inherited trait methods for this.